splunk_connect_for_snmp_poller/manager/poller.py

```diff
@@ -131,8 +133,33 @@ def __check_inventory(self):
                     else:
                         self.update_schedule_for_changed_conf(entry_key, ir, profiles)
 
+            if server_config_modified:
+                new_enricher = self._server_config.get("enricher", {})
+                if new_enricher != self._old_enricher:
+                    self.run_enricher_check(new_enricher, profiles, inventory_hosts)
             self.clean_job_inventory(inventory_entry_keys, inventory_hosts)
 
+    def run_enricher_check(self, new_enricher, profiles, inventory_hosts):
+        logger.debug(
```

Comment:
Maybe as this called on demand we should put info level? What do you think?

Reply:
Good idea, changed